monitor/alerts/model.py

```diff
@@ -41,7 +59,10 @@ def to_slack(self, is_slack_workflow: bool = False) -> SlackMessageSchema:
                                               [f'*Model*\n{self.alias}',
                                                f'*When*\n{self.detected_at}'],
                                               divider=True)
-        self._add_fields_section_to_slack_msg(slack_message, [f'*Owners*\n{self.owners}', f'*Tags*\n{self.tags}'])
+        self._add_fields_section_to_slack_msg(slack_message,
+                                              [f'*Owners*\n{self.owners}', f'*Tags*\n{self.tags}'])
+        if self.subscribers:
+            self._add_fields_section_to_slack_msg(slack_message, [f'*Subscribers*\n{", ".join(set(self.subscribers))}'])
```

Comment:
maybe put it together with column if you have it?

Reply:
Not sure I understand the request 😅

Reply:
Explained over slack, I suggested putting the subscribers and the test column if exists at the same row above the error message

Reply:
It won't work well.
The column field is optional as well + might contain 2 message fields.
Doing this like that makes it messy (need to have if statements that looks at 3 different fields)
Ugly, and don't worth it at the time IMO